hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/SparkYarnRunnerBuilder.java

```diff
@@ -258,10 +258,19 @@ public YarnRunner getYarnRunner(Project project,
         LocalResourceVisibility.APPLICATION.toString(),
         LocalResourceType.FILE.toString(), null), false);
 
+    // Add tf-spark-connector for Featurestore
+    builder.addLocalResource(new LocalResourceDTO(
+        settings.getTfSparkConnectorFilename(), settings.getTfSparkConnectorPath(),
```

Comment:
@Limmen if this file is common for all users, set visibility to FILE

Reply:
There are only public, private and application as possible visibility settings

Reply:
Fixed